server/services/stats_service.py

```diff
@@ -19,32 +24,50 @@
 
 
 class StatsService:
-
     @staticmethod
-    def update_stats_after_task_state_change(project_id: int, user_id: int, last_state: TaskStatus,
-                                             new_state: TaskStatus, action='change'):
+    def update_stats_after_task_state_change(
+        project_id: int,
+        user_id: int,
+        last_state: TaskStatus,
+        new_state: TaskStatus,
+        action="change",
+    ):
         """ Update stats when a task has had a state change """
 
-        if new_state in [TaskStatus.READY, TaskStatus.LOCKED_FOR_VALIDATION, TaskStatus.LOCKED_FOR_MAPPING]:
+        if new_state in [
+            TaskStatus.READY,
```

Comment:
May be we should also include `AUTO_UNLOCK_FOR_MAPPING` and `AUTO_UNLOCK_FOR_VALIDATION` here, since there is no state change

Reply:
Just realised after AUTO UNLOCK  task status is set to READY - so we need not include these here